sp800-63d/vot_mapping.md

```diff
@@ -1,54 +1,82 @@
-<a name="sec1"></a>
+<a name="vectors-of-trust-mapping"></a>
 
 <div class="breaker"></div>
 
-## VOT Overview
+## Mapping Assurance Levels to Vectors of Trust
+_This section is normative._
 
-VOT is comprised of components, and components contain values.
+Vectors of Trust (VoT) is a standard for expressing information about an authentication transaction from an IdP to an RP. This information is split into several orthogonal component categories, much in the same way that the assurances defined in 800-63 are split into different xAL categories.
 
-## Assurance to VOT Mapping
+### Trustmark
 
-_This section is normative._
+The trustmark URI for these definitions is [[ URI for this document ]].
+
+### xAL to VoT Components
+
+This document uses three categories defined in VoT: Identity Proofing (P), Primary Credential Usage (C), and Assertion Presentation (A). These categories correspond to the Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. 
+
+This document does not use the Primary Credential Management (M) category defined in VoT.
+
+This document does not define any additional component categories. Any additional categories SHALL NOT be used.
```

Comment:
If iGov defines more then it'll need its own anchor document. That document is allowed to explicitly inherit values from here. We're saying that if you want to do the 800-63-D flavor of VoT then it comes with these categories and values and none others.